src/include/miopen/solver.hpp

```diff
@@ -799,6 +855,27 @@ struct ConvHipImplicitGemmV4R4GenXdlopsFwdFp32 : SolverBase<ConvolutionContext>
                               float& elapsed_time) const;
 };
 
+struct ConvHipImplicitGemmV4R4WrW : SolverBase<ConvolutionContext>
+{
+    static std::tuple<int, int, int> CalculateGemmSize(const ConvolutionContext& ctx);
+    bool IsApplicable(const ConvolutionContext& ctx) const;
+    PerformanceImplicitGemmV4R4WrW GetPerformanceConfig(const ConvolutionContext& ctx) const;
+    bool IsValidPerformanceConfig(const ConvolutionContext& ctx,
+                                  const PerformanceImplicitGemmV4R4WrW& config) const;
+    PerformanceImplicitGemmV4R4WrW Search(const ConvolutionContext&) const;
+    int RunAndMeasureSolution(miopen::Handle& profile_h,
```

Comment:
https://github.com/ROCmSoftwarePlatform/MIOpen/pull/166/files#r422254802

Reply:
RunAndMeasure? 
But from atam's comment
it's related to PR 203 and it not merge now

Reply:
You are right

Reply:
@asleepzzz @asroy I am sorry for confusion due to my previous posts where I said that we should stop using `RunAndMeasure` immediately. Then I published a refutation, but you might have missed it.

We still have to implement and use `RunAndMeasure` until #203 is committed.

However please note that each new tunable Solver would complicate @DrizztDoUrden's work on #203.

Reply:
Not many complications besides a rather trivial merge. Don't worry, I'll be fine :)